.rubocop_todo.yml

```diff
@@ -0,0 +1,30 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2015-11-10 17:59:46 -0800 using RuboCop version 0.34.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 9
```

Comment:
Some of these seem pretty straightforward to fix (or, at least, push up to the rubocop.yml), aren't they?